oximeter/collector/src/lib.rs

```diff
@@ -11,12 +11,15 @@ use dropshot::ConfigLogging;
 use dropshot::HttpError;
 use dropshot::HttpServer;
 use dropshot::HttpServerStarter;
-use internal_dns::resolver::ResolveError;
-use internal_dns::resolver::Resolver;
 use internal_dns::ServiceName;
+use omicron_common::address::{get_internal_dns_server_addresses, DNS_PORT};
 use omicron_common::api::internal::nexus::ProducerEndpoint;
 use omicron_common::backoff;
 use omicron_common::FileKv;
+use qorb::backend;
+use qorb::resolver::{AllBackends, BoxedResolver, Resolver};
+use qorb::resolvers::dns::{DnsResolver, DnsResolverConfig};
```

Comment:
Style nit: can we split these imports, to match the style for the existing ones?

Reply:
Updated in 834ea859e3a87204b4f370c42ae888c1ca966d22